Formula/gaudi.rb

```diff
@@ -16,20 +16,19 @@ class Gaudi < Formula
   depends_on "gperftools"
   depends_on "heppdt2"
   depends_on "jemalloc"
-  depends_on "python"
+  depends_on "python@3.9"
   depends_on "root"
   depends_on "tbb"
   depends_on "xerces-c"
 
   def install
     # Fix cmake issue with Foundation framework
-    text = File.read("GaudiKernel/CMakeLists.txt")
-    replace = text.gsub(/target_link_libraries\(GaudiKernel PUBLIC Foundation\)/,
-                        "target_link_libraries(GaudiKernel PUBLIC ${Foundation_FRAMEWORK})")
-    File.open("GaudiKernel/CMakeLists.txt", "w") { |file| file.puts replace }
 
+    inreplace("GaudiKernel/CMakeLists.txt", /target_link_libraries\(GaudiKernel PUBLIC Foundation\)/,
+                        "target_link_libraries(GaudiKernel PUBLIC ${Foundation_FRAMEWORK})")
```

Comment:
Does this not work?
```suggestion
    inreplace "GaudiKernel/CMakeLists.txt", /target_link_libraries\(GaudiKernel PUBLIC Foundation\)/,
                        "target_link_libraries(GaudiKernel PUBLIC ${Foundation_FRAMEWORK})"
```

Reply:
Yes, I cross checked that the file is correctly modified and I built again from source without any issues.

Reply:
Do you want me to remove the parenthesis ?

Reply:
Yes. Style in homebrew-core is to avoid parentheses unless necessary.

Reply:
Done